activities/Measure.activity/lib/speechpalette.js

```diff
@@ -0,0 +1,48 @@
+define(["sugar-web/graphics/palette",
+        "text!speechpalette.html"], function (palette, template) {
+
+    'use strict';
+
+    var activitypalette = {};
+
+    activitypalette.ActivityPalette = function (activityButton) {
+
+        palette.Palette.call(this, activityButton);
+
+        var activityTitle;
+        var descriptionLabel;
+        var descriptionBox;
+
+        this.getPalette().id = "activity-palette";
+
+        var containerElem = document.createElement('div');
+        containerElem.innerHTML = template;
+        this.setContent([containerElem]);
+
+        this.pitchScale = containerElem.querySelector('#pitchvalue');
+        this.rateScale = containerElem.querySelector('#ratevalue');
+
+        this.pitchScale.onclick = function() {
+            console.log("he;;p");
```

Comment:
Can you please remove these console messages .

Reply:
fixed